core/src/main/resources/jenkins/model/Jenkins/configure.jelly

```diff
@@ -39,14 +39,6 @@ THE SOFTWARE.
       <f:entry title="${%Home directory}" help="/help/system-config/homeDirectory.html">
         ${it.rootDir}
       </f:entry>
-      <f:advanced>
-        <f:entry field="rawWorkspaceDir" title="${%Workspace Root Directory}">
-          <f:textbox/>
-        </f:entry>
-        <f:entry field="rawBuildsDir" title="${%Build Record Root Directory}">
-          <f:textbox/>
-        </f:entry>
-      </f:advanced>
```

Comment:
So we're removing the readonly UI after all?

Reply:
@daniel-beck I finally took the path Jesse advised in https://github.com/jenkinsci/jenkins/pull/3364#discussion_r176424745

Reply:
Preventing manipulated browser form submissions and not showing the current configuration are not necessarily the same thing.

This could easily be a table showing the current values and perhaps providing example paths.

Reply:
IMO: users should not be customizing this. If they do, they presumably have some reason and will be able to figure out example paths for themselves. If they are not customizing this, then they should have no reason to care. And the workspace root is only relevant for on-master builds, which you should never do to begin with.

Reply:
@jglick 

> And the workspace root is only relevant for on-master builds, which you should never do to begin with.

Jenkins does it for you in matrix parents, pipelines, etc.

ci.j.io gets the path sanitization warning from `jenkins.branch.WorkspaceLocatorImpl locate` regularly due to the legacy setting, and there are no builds executing on master.